test/Microsoft.TestPlatform.TestUtilities/IntegrationTestEnvironment.cs

```diff
@@ -190,15 +190,15 @@ public string RunnerFramework
         /// (c) Name of the test asset matches the parent directory name. E.g. <c>TestAssets\SimpleUnitTest\SimpleUnitTest.xproj</c> must 
         /// produce <c>TestAssets\SimpleUnitTest\bin\Debug\SimpleUnitTest.dll</c>
         /// </remarks>
-        public string GetTestAsset(string assetName)
+        public string GetTestAsset(string assetName, string targetFramework = null)
```

Comment:
To run xunit acceptance test which won't support net 451. Added overload method.